library/create_kafka.py

```diff
@@ -0,0 +1,239 @@
+#!/usr/bin/python
+
+
+from __future__ import (absolute_import, division, print_function)
+from sys import api_version
+__metaclass__ = type
+
+DOCUMENTATION = r'''
+---
+module: create_kafka
+
+short_description: Create Red Hat OpenShift Streams for Apache Kafka Instance
+
+# If this is part of a collection, you need to use semantic versioning,
+# i.e. the version is of the form "2.5.0" and not "2.4".
+version_added: "0.1.0"
+
+description: Create Red Hat OpenShift Streams for Apache Kafka Instance
+
+options:
+    name:
+        description: Name of the Kafka instance
+        required: true
+        type: str
+    cloud_provider:
+        description: Cloud provider for the Kafka instance
+        required: false
+        type: str
+        default: aws
+    region:
+        description: Region of the Kafka instance
+        required: false
+        type: str
+        default: eu-west-1
+    reauthentication_enabled:
+        description: Reauthentication enabled for the Kafka instance
+        required: false
+        type: bool
+        default: true
+    plan:
```

Comment:
[Discussion point] How I can get values of the plan? Calculating plan is not trivial on it's own (it is 900 lines in the CLI)

Reply:
This is a good point and it is not something that I have an answer to. how do we check up that the plans available to the user based on their identity?

Reply:
We need to discuss that with team as terraform will have the same problem. 
Not sure how @jackdelahunt solved that issue. He has kafka creation as release pending so this much have been solved by him in Terraform provider. Let's collab to exchange knowledge